### PR TITLE
Hints and tips, suspicious snippets

### PR DESCRIPTION
**Pull request by tbazant**
* merged @asettle 's recent changes
* 2 comments remain and need to be reviewed

### CODE REVIEW

File: xml/bp_troubleshooting_tips.xml [hunk 1]
```diff
@@ -136,19 +134,16 @@ osd_scrub_load_threshold = 0.25
...
    After OSD services are started, unset the cluster from
    <literal>noout</literal>:
   </para>
 
-<screen>
-&prompt.cephuser;ceph osd unset noout
-</screen>
+<screen>&prompt.cephuser;ceph osd unset noout</screen>
  </sect1>
+<!-- 2020-09-22 tbazant: i suggest removing the following whole section -->
```
Comment (asettle) [inline xml/bp_troubleshooting_tips.xml:146]: @smithfarm what do you think of this suggestion? The following section online is: https://susedoc.github.io/doc-ses/master/html/ses-troubleshooting/storage-tips.html#Cluster-Time-Setting 

Sounds like a reasonable idea to me.

Comment (smithfarm) [reply xml/bp_troubleshooting_tips.xml:146]: :+1:

File: xml/bp_troubleshooting_tips.xml [hunk 2]
```diff
@@ -40,6 +40,7 @@
      journal devices, and compare the output to the previously saved list of
      partitions:
     </para>
+<!-- 2020-09-22 tbazant: the following needs review, path does not exist -->
```
Comment (asettle) [inline xml/bp_troubleshooting_tips.xml:43]: Jan is working on this here: https://github.com/SUSE/doc-ses/pull/597

**Review by asettle — APPROVED** [review]

File: xml/bp_troubleshooting_tips.xml [hunk 3]
```diff
@@ -744,20 +641,25 @@ ID WEIGHT TYPE NAME    UP/DOWN REWEIGHT PRIMARY-AFFINITY
...
   <note>
-    <para>
-      It is also possible that the battery on the motherboard has died, and
-      the clock skew will be more pronounced. If this is the case, be aware
-      that it will take quite some time for chronyd to re-sync the clocks.
-    </para>
+   <para>
+    It is also possible that the battery on the motherboard has died, and the
+    clock skew will be more pronounced. If this is the case, be aware that it
+    will take quite some time for chronyd to re-sync the clocks.
```
Comment (lproven) [inline xml/bp_troubleshooting_tips.xml:653]:
```suggestion
    will take quite some time for <systemitem class="daemon">chronyd</systemitem> to re-synchronize
    the clocks.
```

File: xml/bp_troubleshooting_tips.xml [hunk 4]
```diff
@@ -744,20 +641,25 @@ ID WEIGHT TYPE NAME    UP/DOWN REWEIGHT PRIMARY-AFFINITY
...
+   </para>
   </note>
+
   <para>
-    If you receive a clock skew warning, confirm that the <systemitem class="daemon">chronyd.service</systemitem>
-    is running on all cluster nodes. If not, restart the service and wait for
-    chronyd to re-sync the clock.
+   If you receive a clock skew warning, confirm that the
+   <systemitem class="daemon">chronyd.service</systemitem> is running on all
```
Comment (lproven) [inline xml/bp_troubleshooting_tips.xml:659]:
```suggestion
   <systemitem class="daemon">chronyd.service</systemitem> daemon is running on all
```

Comment (lproven) [reply xml/bp_troubleshooting_tips.xml:659]: Or "service", but since we introduce it with "the" we need some classification here

**Review by lproven — DISMISSED** [review]
Some other suggestions about unchanged, older text.

Line 38
> Run <command>readlink</command> against all block.wal, block.db, and

Should these be in [filename] or [literal] tags?

Line 341
>  You can create a disk image for &kvm;-driven virtual machine, store it in a
For _a_ KVM-driven VM

Line 436
>  Following is a list of &ceph; related services and numbers of the ports that
ceph-related

Line 545
> Intermittent and complete network failures will impact a &ceph; cluster.
_Both_ intermittent and...

Line 553
>  Try the runner <literal>network.ping</literal> to ping between cluster
[command]ping[/command]

Line 773-4
> There are more reasons why the cluster performance may become weak. One of
them can be network problems.

There may be other reasons why cluster performance becomes weak, such as network problems.

Line 804

>  ports/protocols required by &ceph; operation. See

ports or protocols

**Comment by tbazant** [comment]
> There may be other reasons why cluster performance becomes weak, such as network problems.

Your suggestions are addressed.

**Pull request merged**